### PR TITLE
Fix for heartbeat timeout dont get from alert labels

### LINKED ISSUE #1401: heartbeat timeout dont get from alert labels
alert in prometheus
```yaml
groups:
- name: targets
  rules:
  - alert: DeadMansSwitch
    expr: vector(1)
    labels:
      severity: critical
      event: Heartbeat
      timeout: 1800
      monitor: nop_m10_collector
    annotations:
      description: This is a DeadMansSwitch meant to ensure that the entire Alerting pipeline
        is functional.
      summary: Alerting DeadMansSwitch
```

created heartbeat 
```
    {
      "attributes": {
        "environment": "Production", 
        "group": "Prometheus", 
        "service": [
          ""
        ], 
        "severity": "critical"
      }, 
      "createTime": "2020-12-01T21:15:24.964Z", 
      "customer": null, 
      "href": "http://metrics.sys.local/alerta/api/heartbeat/1c1103ba-ac24-4eab-95f6-bc5557d62536", 
      "id": "1c1103ba-ac24-4eab-95f6-bc5557d62536", 
      "latency": 0, 
      "maxLatency": 2000, 
      "origin": "prometheus/nop_m10_collector", 
      "receiveTime": "2020-12-01T21:15:24.964Z", 
      "since": 459, 
      "status": "ok", 
      "tags": [
        "alertname=DeadMansSwitch", 
        "collector=nop_m10_collector", 
        "timeout=1800"
      ], 
      "timeout": 86400, 
      "type": "Heartbeat"
    }
```

### PR DESCRIPTION
**Pull request by proffust**

### CODE REVIEW

File: alerta/plugins/heartbeat.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def pre_receive(self, alert, **kwargs):
         HEARTBEAT_EVENTS = self.get_config('HEARTBEAT_EVENTS', default=['Heartbeat'], type=list, **kwargs)
 
         if alert.event in HEARTBEAT_EVENTS:
-            if 'timeout' in request.json:
+            if 'timeout' in request.json["commonLabels"]:
```
Comment (satterly) [inline alerta/plugins/heartbeat.py:22]: Sorry but I don't see how this fixes the issue. Maybe I'm missing something. If so, please elaborate. Thanks.

Comment (satterly) [reply alerta/plugins/heartbeat.py:22]: If there was a test that showed this worked after the change that would probably be enough for me to merge.

Comment (proffust) [reply alerta/plugins/heartbeat.py:22]: yes, i tested on my productive service.
request.json is `{'receiver': 'alerta', 'status': 'firing', 'alerts': [{'status': 'firing', 'labels': {'alertname': 'DeadMansSwitch', 'collector': 'shm10_collector', 'event': 'Heartbeat', 'monitor': 'shm10_collector', 'severity': 'critical', 'timeout': '1800'}, 'annotations': {'description': 'This is a DeadMansSwitch meant to ensure that the entire Alerting pipeline is functional.', 'summary': 'Alerting DeadMansSwitch'}, 'startsAt': '2020-10-19T15:58:33.772089574Z', 'endsAt': '0001-01-01T00:00:00Z', 'generatorURL': 'http://bfdf51e9472c:9090/graph?g0.expr=vector%281%29&g0.tab=1'}], 'groupLabels': {'alertname': 'DeadMansSwitch'}, 'commonLabels': {'alertname': 'DeadMansSwitch', 'collector': 'shm10_collector', 'event': 'Heartbeat', 'monitor': 'shm10_collector', 'severity': 'critical', 'timeout': '1800'}, 'commonAnnotations': {'description': 'This is a DeadMansSwitch meant to ensure that the entire Alerting pipeline is functional.', 'summary': 'Alerting DeadMansSwitch'}, 'externalURL': 'http://42e370a7a6f5:9093', 'version': '4', 'groupKey': '{}:{alertname="DeadMansSwitch"}'}`
this dict not included timeout key. commonLabels field contain all keys

Comment (satterly) [reply alerta/plugins/heartbeat.py:22]: This fix needs to go into the Prometheus webhook not the heartbeats plugin as it's specific to Prometheus Alertmanager.

**Pull request closed without merging**